src/utils/systemapi.ts

```diff
@@ -0,0 +1,453 @@
+import { WaziGateApiUrl } from '../constants/index.ts'; 
```

Comment:
I find strange that we have a "waziup" library, but not for system APIs. Any suggestion?

Reply:
It's possible to have the system api inside the waziup library. Because it's just a wrapper on it. We can discuss more with the team on this.